docs/source/usage/cli.rst

```diff
@@ -85,7 +85,7 @@ The ``create-meta`` command can be used as follows:
 
 .. code-block:: bash
 
-   metasyn create-meta [input] [output]
+   ``metasyn`` create-meta [input] [output]
```

Comment:
I think I would prefer plain metasyn in the code blocks.

Reply:
woops. I was so careful with find and replacing, but not careful enough.